exception.company.duplicate_name_email=Ошибка!<br/>Компания с таким названием и email уже есть в приложении!
exception.users.duplicate_email=Ошибка!<br/>Пользователь с такой почтой уже есть в приложении!
user.save=Пользователь успешно сохранен